### PR TITLE
Release/v0.25.0

### PR DESCRIPTION
**Pull request by ebuchman**
v0.25.0!

### CODE REVIEW

**Comment by ebuchman** [comment]
We should improve https://github.com/tendermint/tendermint/blob/release/v0.25.0/docs/introduction/install.md#compile-with-cleveldb-support. Can we remove the `sudo` 's there? We should also add instructions for mac. And we might want to mention this in the `running in production` docs, that goleveldb has issues.

**Comment by ebuchman** [comment]
Also need to document the boundary conditions for the ConsensusParams and possibly unify them (ie. right now MaxGas >= -1 but EvidenceAge > 0.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
 # Changelog
 
+## v0.25.0
+
+*September 22, 2018*
```
Comment (xla) [inline CHANGELOG.md]: Can we use ISO 8601 as recommended byt he Government of Canada?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Or [Blockchain](https://en.wikipedia.org/wiki/Blockchain_(database)) for short.
 [![API Reference](
 https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667
 )](https://godoc.org/github.com/tendermint/tendermint)
-[![Go version](https://img.shields.io/badge/go-1.9.2-blue.svg)](https://github.com/moovweb/gvm)
+[![Go version](https://img.shields.io/badge/go-1.10.4-blue.svg)](https://github.com/moovweb/gvm)
```
Comment (xla) [inline README.md]: Why not `1.11`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+
+FEATURES:
+- [rpc] [\#2415](https://github.com/tendermint/tendermint/issues/2415) New `/consensus_params?height=X` endpoint to query the consensus
+  params at any height (@scriptonist)
+- [types] [\#1714](https://github.com/tendermint/tendermint/issues/1714) Add Address to GenesisValidator
+- [metrics] [\#2337](https://github.com/tendermint/tendermint/issues/2337) `consensus.block_interval_metrics` is now gauge, not histogram (you will be able to see spikes, if any)
+
+IMPROVEMENTS:
+- [libs/db] [\#2371](https://github.com/tendermint/tendermint/issues/2371) Output error instead of panic when the given `db_backend` is not initialised (@bradyjoestar)
+- [libs] [\#2286](https://github.com/tendermint/tendermint/issues/2286) Enforce 0600 permissions on `autofile` and `db/fsdb`
```
Comment (melekes) [inline CHANGELOG.md]: `Panic if permissions change`. I don't think it's the same as `Enforce ...`

**Pull request merged**